### PR TITLE
added case study of data collection for rural water systems in Rwanda

### PR DESCRIPTION
**Pull request by JinIgarashi**
@m-kuhn I drafted a case study of data collection by QField in WASAC, Rwanda. I have tested locally, please check and adjust the layout if necessary. Also, some of English grammar or words might be wrong, so please correct it.

### CODE REVIEW

File: en/case-studies/rwanda-rural-water.rst [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+To conduct data collection of all rural water supply network in Rwanda, 
+and keep updating the data continuously in order to improve operation & 
+maintenance of waterworks.
+
+Project preparation
+-------------------
+
+Before starting our data collection, we conducted the following things.
+ - Develop our own PostGIS database
+ - Develop QGIS project template with Geopackage. The Geopackage table design should be equal to PostGIS. So we can just copy and paste to PostGIS.
```
Comment (m-kuhn) [inline en/case-studies/rwanda-rural-water.rst:18]:
```suggestion
 - Develop QGIS project template with Geopackage. The Geopackage table design is equal to PostGIS to be able to copy and paste to PostGIS.
```

File: en/case-studies/rwanda-rural-water.rst [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+Apart from preparing Android devices, if possible, you may purchase GPS devices. In WASAC, we bought Garmin GPSMAP 64S.
```
Comment (m-kuhn) [inline en/case-studies/rwanda-rural-water.rst:20]:
```suggestion
Apart from preparing Android devices, we purchased GPS devices for higher positioning accuracy. In WASAC, we bought Garmin GPSMAP 64S. 
```

File: en/case-studies/rwanda-rural-water.rst [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+Apart from preparing Android devices, if possible, you may purchase GPS devices. In WASAC, we bought Garmin GPSMAP 64S. 
+Sometimes, GPS of smartphone and tablet is not very accurate, so we normally capture the same location by using Garmin GPS, then correct the location of QField's data after data collection work.
+
+Data collection
+---------------
+
+Once we prepared Geopackage and QGIS project template, 
+we conducted training of QGIS/QField in July 2018 and 
+launched our data collection work in 27 districts in the whole country of Rwanda. 
+27 engineers sent their Geopackage to central office in Kigali,
```
Comment (m-kuhn) [inline en/case-studies/rwanda-rural-water.rst:29]:
```suggestion
27 engineers sent their Geopackage to the central office in Kigali, 
```

File: en/case-studies/rwanda-rural-water.rst [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+Sometimes, GPS of smartphone and tablet is not very accurate, so we normally capture the same location by using Garmin GPS, then correct the location of QField's data after data collection work.
+
+Data collection
+---------------
+
+Once we prepared Geopackage and QGIS project template, 
+we conducted training of QGIS/QField in July 2018 and 
+launched our data collection work in 27 districts in the whole country of Rwanda. 
+27 engineers sent their Geopackage to central office in Kigali, 
+the MIS specialist validated and entered their data from Geopackage to PostGIS database.
```
Comment (m-kuhn) [inline en/case-studies/rwanda-rural-water.rst:30]: What is the "MIS specialist"? Has this been introduced before?

File: en/case-studies/rwanda-rural-water.rst [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+   .. image:: /images/rwanda-rural-water-1.png
+      :width: 600px
+      :alt: data collection procedure
+
+Data distribution and updating
+------------------------------
+
+The most significant thing after data collection is ``updating``. 
+We have seen many organization in Africa, they normally failed to keep updating.
```
Comment (m-kuhn) [inline en/case-studies/rwanda-rural-water.rst:43]:
```suggestion
We have seen many organization in Africa, which failed to keep data up to date.
```

File: en/case-studies/rwanda-rural-water.rst [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+Data distribution and updating
+------------------------------
+
+The most significant thing after data collection is ``updating``. 
+We have seen many organization in Africa, they normally failed to keep updating. 
+Several years later, their data will normally become too old, 
+and most of them need to put efforts on data collection again. 
+
+However, WASAC is continuously updating all of the data until now.
```
Comment (m-kuhn) [inline en/case-studies/rwanda-rural-water.rst:47]:
```suggestion
WASAC decided to continuously update all of the data and keeps doing this until now. 
```

File: en/case-studies/rwanda-rural-water.rst [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+Data distribution and updating
+------------------------------
+
+The most significant thing after data collection is ``updating``. 
+We have seen many organization in Africa, they normally failed to keep updating. 
+Several years later, their data will normally become too old, 
+and most of them need to put efforts on data collection again. 
+
+However, WASAC is continuously updating all of the data until now. 
+We think QField works very well for this purpose.
```
Comment (m-kuhn) [inline en/case-studies/rwanda-rural-water.rst:48]:
```suggestion
QField has proven to be very well suited for this purpose.
```

File: en/case-studies/rwanda-rural-water.rst [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+
+In order to distribute and updating the data, 
+we developed a python script `postgis2qfield <https://github.com/WASAC/postgis2qfield>`__. 
+This ``postgis2qfield`` tool can extract the data from PostGIS and create Geopackages for each districts in Rwanda.
```
Comment (m-kuhn) [inline en/case-studies/rwanda-rural-water.rst:52]:
```suggestion
This ``postgis2qfield`` tool can extract the data from PostGIS and create Geopackages for each district in Rwanda. 
```

File: en/case-studies/rwanda-rural-water.rst [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+
+However, WASAC is continuously updating all of the data until now. 
+We think QField works very well for this purpose.
+
+In order to distribute and updating the data, 
+we developed a python script `postgis2qfield <https://github.com/WASAC/postgis2qfield>`__. 
+This ``postgis2qfield`` tool can extract the data from PostGIS and create Geopackages for each districts in Rwanda. 
+We upload these 27 geopackage together with QGIS project template to Google Drive storage. 
+After that, those engineers in districts download their geopakage to Android device to continue adding and updating the data. 
+Once they completed updating, send geopackage to central office again, MIS specialist update PostGIS database and regenerate geopackages for QField.
```
Comment (m-kuhn) [inline en/case-studies/rwanda-rural-water.rst:55]:
```suggestion
Once they completed updating, they sent the geopackage to central office again, MIS specialist update PostGIS database and regenerate geopackages for QField.
```

File: en/case-studies/rwanda-rural-water.rst [hunk 10]
```diff
@@ -0,0 +1,111 @@
...
+.. container:: clearer text-center
+
+   .. image:: /images/rwanda-rural-water-2.png
+      :width: 600px
+      :alt: data distribution and updating procedure
+
+Why didn't we use ``QFieldSync`` plugin for this purpose? 
+
+Although QField has a nice QFieldSync plugin for linking between PostGIS and Geopackage, 
+we didn't select QFieldSync. Because QFieldSync plugin only can create a Geopackage only,
```
Comment (m-kuhn) [inline en/case-studies/rwanda-rural-water.rst:66]: That's actually not completely true, QFieldSync can prepare a subset of the data only. But it's always a bounding box / rectangular.

Comment (m-kuhn) [reply en/case-studies/rwanda-rural-water.rst:66]: I wonder if we could just remove this paragraph? It highlights drawbacks of the QFieldSync plugin. One the one hand it's not completely correct (see above) and they will likely be inappropriate in the near future and start to be misleading for people reading this.

**Review by m-kuhn — COMMENTED** [review]
Very nice case study, thanks a lot for providing this !

I have left a couple of comments for your condsideration, please let me know what you think.

File: en/case-studies/rwanda-rural-water.rst (hunk 4 above)
Comment (JinIgarashi) [reply en/case-studies/rwanda-rural-water.rst:30]: It is a position in WASAC. MIS (Management Information System) Specialist. Let me add it.

File: en/case-studies/rwanda-rural-water.rst (hunk 10 above)
Comment (JinIgarashi) [reply en/case-studies/rwanda-rural-water.rst:66]: > But it's always a bounding box / rectangular

Sure, I remember QFieldSync has this function. However, in WASAC, they need to separate data exactly by the boundary of district. Also, it is not easy for them to repeat the same process for all the districts. That is why I developed customized python script.

> I wonder if we could just remove this paragraph? 

Yeah, I think I can remove this paragraph.

File: en/case-studies/rwanda-rural-water.rst (hunk 4 above)
Comment (JinIgarashi) [reply en/case-studies/rwanda-rural-water.rst]: I added description of MIS in this commit(https://github.com/opengisch/QField-docs/pull/77/commits/5d15d0943a202e52235bb906d356519148aa2030).

File: en/case-studies/rwanda-rural-water.rst (hunk 10 above)
Comment (JinIgarashi) [reply en/case-studies/rwanda-rural-water.rst]: I deleted the paragraph in this commit(https://github.com/opengisch/QField-docs/pull/77/commits/088ffee04cd7ee0a23b4d85e222b66ddb00452eb).

**Comment by JinIgarashi** [comment]
Thank you for reviewing. I modified text (I left comments in the above). Could you check them?

**Comment by m-kuhn** [comment]
Thank you very much!

@lucienicolier can you give this a final review and merge if it is good?

**Review by lucienicolier — APPROVED** [review]
Nice case study, I like how it's explained how much it's important to maintain the data up to date.

Thank you @JinIgarashi

**Pull request merged**